luigi_pipeline/lib/model/sv_mt_schema.py

```diff
@@ -226,9 +226,17 @@ def _genotype_fields(self):
             'num_alt': hl.if_else(is_called, self.mt.GT.n_alt_alleles(), -1)
         }
 
+    def _genotype_filter_samples(self, filter):
```

Comment:
Worth acknowledging that the return type changed from an array (in `load_data.py`) to a `set` here to match the other genotype histograms.  Not sure if that matters.